docs/quick-reference.adoc

```diff
@@ -1,2 +1,215 @@
 = Sylt Quick Reference
 :favicon: sylt.png
+
+Here you will find most things in Sylt in a concise format.
+
+.Variables
+[source, sylt]
+----
+constant :: 3.14
+variable := []
+two: int : 2
+text: str = "abc"
+text = "A short sentence."  // <1>
+----
+
+<1> A single `=` is used to set the value of a variable after declaration with
+`:=`.
+
+.If-else statements
+[source, sylt]
+----
+if 1 < 0 do
+    // ...
+else if "hello" == "" do
+    // ...
+else do
+    // ...
+end
+
+one :: 1 if true else 2
+two :: one + 1 if == 2 else -1 // <1>
+----
+
+<1> The left hand side is inserted right after `if`
```

Comment:
Maybe say what it can be expanded to? I'm not 100% sure what you mean by this.

Reply:
The guide covers this, which probably means the link to it should be here.